src/serial.rs

```diff
@@ -586,6 +606,20 @@ mod tests {
         fn tx_lost_byte(&self) {
             self.tx_lost_byte_count.inc();
         }
+
+        fn in_buffer_empty(&self) {
+            match self
+                .buffer_ready_event
+                .as_ref()
+                .map_or(Ok(()), |buf_ready| buf_ready.write(1))
+            {
+                Ok(_) => (),
+                Err(err) => println!(
+                    "Could not signal that serial device buffer is ready: {:?}",
+                    err
+                ),
```

Comment:
In tests we should use unwraps because otherwise we might end up with silent fails.